big-practice/src/js/helpers/show-message.helper.js

```diff
@@ -0,0 +1,13 @@
+export class ShowMessage {
```

Comment:
Class should be Noun. Ex: Document class. Document.querySelector()

Reply:
updated!